ddtrace/contrib/openai/_metrics.py

```diff
@@ -0,0 +1,26 @@
+"""
+Module providing a stats client to be used by the OpenAI integration.
+"""
+from ddtrace import config
+from ddtrace.internal.agent import get_stats_url
+from ddtrace.internal.dogstatsd import get_dogstatsd_client
+
+
+_statsd = None
+
+
+def stats_client():
+    global _statsd
+    if _statsd is None and config.openai.metrics_enabled:
+        # FIXME: this currently does not consider if the tracer is configured to
```

Comment:
lower priority, might be able to cover it with the metrics code refactor

@Kyle-Verhoog to own